pkg/apic/servicebody.go

```diff
@@ -117,6 +117,9 @@ func (s *ServiceBody) GetAccessRequestDefinition() *management.AccessRequestDefi
 }
 
 func (s *ServiceBody) createAccessRequestDefinition() error {
+	if !s.uniqueARD {
+		return nil
```

Comment:
can you log s.ardName  before you return since we are bailing right out

Reply:
Thinking debug level? Since this would happen a lot I think Warn and Info would not apply. More so since this only happens when the user said the ard is not unique for this api.